### PR TITLE
Classes

### PR DESCRIPTION
**Pull request by julzedz**
In this project, I did the following:
- Created a Person Class.
- Created a Student Class.
- Created a Teacher Class.
- Save each Class in a separate file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,171 @@
...
+
+I would like to thank the [Microverse team](https://www.microverse.org/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+# 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] What you have at the root of your project is the `LICENSE` file. Kindly change the name in this path, or the file in the root 👍

File: student.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
+    super
```
Comment (Shedrack-Sunday) [inline student.rb]: - [ ] Kindly add the properties of the initialize method of the person class in the `super`
- Therefore, the updated Student class includes the inherited instance variables from the Person class (@id, @age, and @name) as well as its own instance variable @classroom..

### Sample:
``super(age, name, parent_permission)``

File: teacher.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super
```
Comment (Shedrack-Sunday) [inline teacher.rb]: - [ ] Kindly make the same changes as in the student class i pointed which was very detailed 👍

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @julzedz     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️  
- Nice work with the use of ruby syntax✔️ 
- Good project setup ✔️ 
  
# Required Changes  🌵 

- [ ] You made some impressive implementations which I must say is a job well done. However, there are a few  I suggest   you make In line with the requirement

- [ ] Add the "Key feature" section of your README with content. it can't be left blank 👍 
- [ ] Remove the live demo since you don't have it

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by roshan-bajgain — APPROVED** [review]
Hi @julzedz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**